app/models/vacation.rb

```diff
@@ -0,0 +1,17 @@
+class Vacation < ApplicationRecord
+
+  belongs_to :employee
+  
+  validates :start_date, presence: true
+  validates :end_date, presence: true
+  validate :end_date_after_start_date
+
+  private
+
+  def end_date_after_start_date
+   if end_date < start_date
```

Comment:
In Ruby we prefer to write it like this:
```ruby
def end_date_after_start_date
  return if end_date > start_date

  errors.add(:end_date, "must be after the start date")
end
```

It's read easier and without nested blocks.